server/core/middlewares/rate-limiter.ts

```diff
@@ -54,6 +55,8 @@ export const activityPubRateLimiter = buildRateLimiter({
 // Private
 // ---------------------------------------------------------------------------
 
-function sendRateLimited (res: express.Response, options: RateLimitHandlerOptions) {
+function sendRateLimited (req: express.Request, res: express.Response, options: RateLimitHandlerOptions) {
+  logger.info('Rate limit exceeded for route ' + req.path)
```

Comment:
Maybe in `.debug` mode to prevent logging too much information? Also use `req.originalUrl` to get full path